### DISCUSSION TITLE
Github Student Pack renewal while study abroad

### DISCUSSION THREAD

**Discussion by merlin-mallory** [community/community#66939] [GitHub Education]
### Hi everyone,

I’m having the following problems with my Github student pack renewal:

1. The application form only allows me to use my front camera to take photos of my proof of identification. And Github’s software distorts the image to be a very compressed landscape mode photo, even when my camera is in portrait mode. There is no upload option. There are dropdowns for higher quality back camera options, but they don’t work (I’m using iphone 14 pro but the image is still low quality because of Github’s compression). I’m trying to use my school transcript as proof but it’s impossible, because either the photo will be landscape distorted, or my hand will be in the photo.

2. I’m studying at Oregon State University in USA, but I’m studying abroad in Japan. My program (OSU postbacc) is 100% remote, but my school transcript does not explicitly state this, for obvious reasons. I do not have a student id because I’m doing the degree remotely while working, so I need to use my transcript, but its not working. The automated system is giving me a lot of bad feedback like my school name is not listed (it is), my school does not have a campus at my browser location (obviously), my transcript does not indicate that I’m remote (my entrire program is remote but it doesn’t explicitly say that on my transcript), etc. If I could upload my multipage transcript then this would all become clear, but I can only take a single poor quality cameraphone photo now, which is ridiculous.

3. I’ve also tried submitting enrollment verification through national clearing house, but the automatic filter is asking me to select “other” as my verification document. But there is no option to do this on the photo-taking page.

How can I fix these issues? Is it possible to make a ticket somewhere else? It doesn’t look like I can submit unless I pass this automatic testing of the photo. I’m just looking to renew my student pack, I’ve already passed the school verification before.

**Comment by Khalidelaasry** [comment]
Hi there,

I'm sorry to hear about the difficulties you're facing with the Github student pack renewal process. It sounds like you've encountered several frustrating issues, especially with the photo verification process.

Given your unique situation of studying remotely abroad, it's understandable that the standard process might not align perfectly with your circumstances. Here's a suggestion on how to proceed:

Contact Github Support: I recommend reaching out to Github Support directly. Explain your situation in detail, just as you did here. Emphasize that your program is 100% remote and that your transcript may not explicitly state this. They might be able to provide alternative solutions or guide you through a different verification process.

Provide Additional Documentation: If possible, try to gather any additional documentation that supports your remote status, such as emails from professors or program administrators acknowledging the remote nature of your studies. This could be useful in explaining your situation to Github.

Request Guidance from OSU: Contact Oregon State University's administration or student services department. Explain your situation and ask if they can provide any supplementary documents or a letter confirming your remote enrollment.

Check for Alternative Verification Methods: Inquire with Github if there are alternative methods of verification available for cases like yours. They may have a manual process or additional options for situations not covered by the automated system.

Patience and Persistence: Dealing with automated systems can be frustrating, but it's important to be persistent. Keep reaching out to Github Support, explaining your situation clearly and providing any additional information they request.

Remember to maintain a polite and patient tone in your communications, as this will likely result in a more positive response from the support team.

I hope this helps you in resolving the issue and successfully renewing your Github student pack. If you have any further questions or need more advice, feel free to ask. Good luck!

**Reply by eleven011-bbs** [reply]
good one

**Comment by Khalidelaasry** [comment]
thanks

**Reply by eleven011-bbs** [reply]
your welcome
